BlockSettleUILib/AuthAddressDialog.cpp

```diff
@@ -236,12 +236,12 @@ void AuthAddressDialog::copySelectedToClipboard()
 
 void AuthAddressDialog::onAddressStateChanged(const QString &addr, const QString &state)
 {
-   if (state == QLatin1String("Verified")) {
+   if (state == QStringLiteral("Verified")) {
```

Comment:
Not related to this PR directly, but I think we shouldn't compare strings for state. Another question is: was this state translated previously? I.e. are we not comparing latin1 with translated strings?

Reply:
That is some old stuff that I accidentally found and I don't want to touch that. Have no idea who made that initially. That is something that we defietely should remove.
So here is [JIRA for that](http://185.213.153.35:8081/browse/BST-2779)